src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V66/Eth66ProtocolHandler.cs

```diff
@@ -160,10 +170,15 @@ private void Handle(GetBlockBodiesMessage getBlockBodies)
 
         private void Handle(GetPooledTransactionsMessage getPooledTransactions)
         {
-            using ArrayPoolList<Transaction> txsToSend = new(1024);
+            _queue.Enqueue(() =>
+            {
+                using ArrayPoolList<Transaction> txsToSend = new(1024);
 
-            Send(new PooledTransactionsMessage(getPooledTransactions.RequestId,
-                FulfillPooledTransactionsRequest(getPooledTransactions.EthMessage, txsToSend)));
+                Send(new PooledTransactionsMessage(getPooledTransactions.RequestId,
+                    FulfillPooledTransactionsRequest(getPooledTransactions.EthMessage, txsToSend)));
+
+                return Task.CompletedTask;
+            });
```

Comment:
By using a `ThrottledActionQueue`, we ensure that we're sending at most one message every `throttleTime`. Note that his behavior is by default disabled and it's only used when sending `PooledTransactionMessage`.